src/main/resources/assets/eln/lang/en_US.lang

```diff
@@ -431,7 +431,7 @@ Player_crankable_shaft=Player crankable shaft
 Can_rotate_slowly=Can rotate slowly
 
 # ./src/main/java/mods/eln/mechanical/Generator.kt
-Converts_mechanical_energy_into_electricity,_or_(badly)_vice_versa.=将机械能转换为电能，或反之(低效)。
+Converts_mechanical_energy_into_electricity,_or_(badly)_vice_versa.=Converts_mechanical_energy_into_electricity,_or_(badly)_vice_versa.
```

Comment:
Shouldn't the later half of this actually have the spaces and such in it rather than _, like the entries below?